support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -1232,6 +1232,19 @@ export function CheckoutComponent({
 								})}
 							</RadioGroup>
 						</FormSection>
+						<div
+							css={css`
+								margin: ${space[6]}px 0;
+							`}
+						>
+							{abParticipations.similarProductsConsent === 'VariantB' && (
+								<Checkbox
+									name="similarProductsConsent"
+									label="Receive information on our products and ways to support and enjoy our journalism. Untick to opt out."
+									checked={true}
+								/>
```

Comment:
Presume visual only, not linked to anything yet (`onChange`)?

Reply:
Ah, don't worry,  I see you get component labeled `similarProductConsent` if `on` the FormOnSubmit